.github/workflows/goreleaser.yml

```diff
@@ -33,4 +33,14 @@ jobs:
         env:
           # Use separate access token, because we need a scope:repo token to publish the brew formula.
           GITHUB_TOKEN: ${{ secrets.GH_PAT }}
- 
+  npm:
+    runs-on: ubuntu-latest
```

Comment:
Should this depend on the goreleaser step? Otherwise there will be a short moment in which it doesn’t work.

Reply:
Good point. I updated the flow to depend on the goreleaser job.